### PR TITLE
New package: riff-2.16.2

### LINKED ISSUE #36885: package request: riff
https://github.com/walles/riff
`riff` is a `git diff` utility to highlight which parts of the line has changed.
screenshot

### PR DESCRIPTION
**Pull request by 0x5c**
Fixes #36885

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/riff/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+revision=1
+build_style=cargo
+short_desc="Diff filter highlighting which line parts have changed"
+maintainer="0x5c <dev@0x5c.io>"
+license="MIT"
+homepage="https://github.com/walles/riff"
+distfiles="https://github.com/walles/riff/archive/refs/tags/${version}.tar.gz"
+checksum=3f7c8e6f2602b97849422f8ad3e1c9fec7fc8cbaf7e7a8ea0856ee8a9cfe7091
+
+
```
Comment (classabbyamp) [inline srcpkgs/riff/template:13]: bonus line here

**Review by classabbyamp — COMMENTED** [review]
tested locally & works well

**Comment by 0x5c** [comment]
ahhh damn, forgot a thing I did for debugging

**Comment by 0x5c** [comment]
Upstream released a new version with all the fixes and better versioning scheme, so I'll fix that before this gets merged

**Comment by 0x5c** [comment]
This should ready for merge

**Pull request merged**